diff --git a/m4/ax_lapack.m4 b/m4/ax_lapack.m4
index 6aa16aa..f4cfe08 100644
--- a/m4/ax_lapack.m4
+++ b/m4/ax_lapack.m4
@@ -109,7 +109,7 @@ if test $ax_lapack_ok = no; then
 fi
 
 # Generic LAPACK library?
-for lapack in lapack lapack_rs6k; do
+for lapack in lapack lapack_rs6k reflapack; do
         if test $ax_lapack_ok = no; then
                 save_LIBS="$LIBS"; LIBS="$BLAS_LIBS $LIBS"
                 AC_CHECK_LIB($lapack, $cheev,
